public/about.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html><html lang="en"><head>
+<!DOCTYPE html><html lang="en"><head></head><body>f
```

Comment:
<head> shouldn't be closed here.  <body>f also looks weird.

Reply:
weird. I fixed it here: 249d56c22475755e7aef6362cccb324432e06a37